dashboard/app/models/survey_result.rb

```diff
@@ -0,0 +1,32 @@
+# == Schema Information
+#
+# Table name: survey_results
+#
+#  id         :integer          not null, primary key
+#  user_id    :integer
+#  properties :text(65535)
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_survey_results_on_user_id  (user_id)
+#
+
+class SurveyResult < ActiveRecord::Base
+  include SerializedProperties
+
+  ETHNICITIES = {}
+  ETHNICITIES["american_indian"] =  "American Indian or Alaska Native"
+  ETHNICITIES["asian"] = "Asian"
+  ETHNICITIES["black"] = "Black or African American"
+  ETHNICITIES["hispanic"] = "Hispanic or Latino"
+  ETHNICITIES["native"] = "Native Hawaiian or Other Pacific Islander "
+  ETHNICITIES["white"] = "White"
+
```

Comment:
you can use `ETHNICITIES.freeze` after you are done modifying it to make the hash unmodifiable (making it a constant makes the reference to the hash constant but does not make the hash unmodifiable)